cmd/agent/main.go

```diff
@@ -81,7 +79,8 @@ func main() {
 				log.Printf("[WARN] [main,podman] [message: Unable to retrieve local agent IP address, using '%s' instead] [error: %s]", options.AgentServerAddr, err)
 				advertiseAddr = options.AgentServerAddr
 			} else {
-				log.Fatalf("[ERROR] [main,docker] [message: Unable to retrieve local agent IP address] [error: %s]", err)
+				log.Printf("[ERROR] [main,docker] [message: Unable to retrieve local agent IP address] [error: %s]", err)
+				advertiseAddr = options.AgentServerAddr
```

Comment:
Same comment as in https://github.com/portainer/agent/pull/228#discussion_r790474656, should this be part of that PR?